openwisp_radius/base/models.py

```diff
@@ -1317,11 +1312,6 @@ def _clean_sms_message(self):
                     )
                 }
             )
-        if (
-            not self.sms_message
-            or self.sms_message == app_settings.SMS_MESSAGE_TEMPLATE
-        ):
-            self.sms_message = None
```

Comment:
Why is this here!? I am getting super confused.

Reply:
This PR was based on #433 earlier, hence these changes appeared here.